Project.toml

```diff
@@ -12,6 +12,7 @@ LightGraphs = "093fc24a-ae57-5d10-9952-331d41423f4d"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
+SparsityDetection = "684fba80-ace3-11e9-3d08-3bc7ed6f96df"
```

Comment:
Why do you need SparsityDetection? And why in this PR?

Reply:
Hey thanks for pointing it out. I added it at some point because I thought its absence was breaking my tests (it wasn't) and forgot to remove it from the list of dependencies afterward. 
Fixed.